src/main/java/games/strategy/internal/persistence/serializable/VersionPersistenceDelegate.java

```diff
@@ -0,0 +1,22 @@
+package games.strategy.internal.persistence.serializable;
+
+import java.io.IOException;
+
+import games.strategy.persistence.serializable.AbstractPersistenceDelegate;
+import games.strategy.util.Version;
+
+/**
+ * A persistence delegate for the {@code Version} class.
+ *
+ * <p>
+ * Instances of this class are immutable.
+ * </p>
+ */
+public final class VersionPersistenceDelegate extends AbstractPersistenceDelegate {
```

Comment:
@DanVanAtta Please take a look at 7e79f75, and let me know if that's what you were thinking of.